### PR TITLE
Activities/iconmeny

### PR DESCRIPTION
**Pull request by malin-mallan**
Startscreen for activityscreen. 

Limitation:
Currently not connected to storage.

### CODE REVIEW

File: src/components/IconMeny.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  "run",
+  "gamepad-variant",
+  "chat",
+  "car"
+];
+
+
+export const IconMeny = (props: any) => {
+  let iconSize = 60;
```
Comment (Neathan) [inline src/components/IconMeny.tsx]: Should probably be standardised through the PaperProvider theme.

File: src/components/IconMeny.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  return (
+    <View style={{ flexGrow: 1, flexDirection: 'row', alignItems: 'stretch', justifyContent: 'center', paddingRight: 20, paddingLeft: 20 }}>
+      <View style={{ flex: 1, justifyContent: 'space-evenly', alignItems: 'center' }}>
+        <IconButton icon={icons[0]} size={iconSize} onPress={() => { props.pressCallback(0, icons[0]) }} />
+        <IconButton icon={icons[1]} size={iconSize} onPress={() => { props.pressCallback(1, icons[1]) }} />
+        <IconButton icon={icons[2]} size={iconSize} onPress={() => { props.pressCallback(2, icons[2]) }} />
+        <IconButton icon={icons[3]} size={iconSize} onPress={() => { props.pressCallback(3, icons[3]) }} />
```
Comment (EMattfolk) [inline src/components/IconMeny.tsx]:
```suggestion
        <IconButton icon={icons[0 + 0 * 4]} size={iconSize} onPress={() => { props.pressCallback(0, icons[0 + 0 * 4]) }} />
        <IconButton icon={icons[1 + 0 * 4]} size={iconSize} onPress={() => { props.pressCallback(1, icons[1 + 0 * 4]) }} />
        <IconButton icon={icons[2 + 0 * 4]} size={iconSize} onPress={() => { props.pressCallback(2, icons[2 + 0 * 4]) }} />
        <IconButton icon={icons[3 + 0 * 4]} size={iconSize} onPress={() => { props.pressCallback(3, icons[3 + 0 * 4]) }} />
```

File: src/components/IconMeny.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { View } from 'react-native';
+import { Text, Button, IconButton, Colors, } from 'react-native-paper';
```
Comment (Neathan) [inline src/components/IconMeny.tsx]:
```suggestion
import React from 'react';
import { View } from 'react-native';
import { IconButton } from 'react-native-paper';

```

**Review by EMattfolk — APPROVED** [review]
Looks good so far!

**Review by LukasJonsson99 — APPROVED** [review]
Looks good!

**Pull request merged**